prep-javascript-variables/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    var fullName = 'Mary Nguyen';
+    console.log('fullName:', fullName);
+    const numberOfPets = 0;
+    console.log('numberOfPets:', numberOfPets);
+    const LikesVegetables = true;
+    Boolean(LikesVegetables);
+    console.log('LikesVegetables:', LikesVegetables);
+    var nothing = 'null';
+    console.log('nothing:', nothing);
+    var never = 'undefined';
```

Comment:
Hi Mary, be sure to leave off the quotes from `undefined` and `null` so that the browser doesn't interpret them as strings.